### DISCUSSION TITLE
Name for "fallback: true"?

### DISCUSSION THREAD

**Discussion by borekb** [vercel/next.js#11123] [Help]
It's a very [clever feature of Next.js 9.3](https://nextjs.org/docs/basic-features/data-fetching#when-is-fallback-true-useful) and I was wondering if it has a more catchy name than "fallback: true".

It's like "postponed / deferred SSG" to me.

Very cool concept BTW!

**Comment by timneutkens** [comment]
It's just `fallback: true` as that's what it does, it enables fallback behavior 👍
